deepinv/tests/test_noise_model.py

```diff
@@ -78,6 +97,30 @@ def test_rng(name, device, rng, dtype):
     assert not torch.allclose(y_1, y_2)
 
 
+@pytest.mark.parametrize("name", NO_RNG_NOISES)
+@pytest.mark.parametrize("device", DEVICES)
+@pytest.mark.parametrize("dtype", DTYPES)
+def test_stats(name, device, dtype):
```

Comment:
```suggestion
def test_noise_moments(name, device, dtype):
```

Slightly longer/descriptive